build/azure-pipelines/common/computeNodeModulesCacheKey.ts

```diff
@@ -0,0 +1,51 @@
+/*---------------------------------------------------------------------------------------------
+  *  Copyright (c) Microsoft Corporation. All rights reserved.
+  *  Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
the file header is incorrect, please copy the header from another file, it should be under the `SOURCE EULA`.